main.go

```diff
@@ -77,10 +78,16 @@ func main() {
 	configLog.EncodeTime = func(ts time.Time, encoder zapcore.PrimitiveArrayEncoder) {
 		encoder.AppendString(ts.UTC().Format(time.RFC3339))
 	}
+	configLog.EncodeLevel = zapcore.CapitalColorLevelEncoder
```

Comment:
it added the colors, but the terminal is not rendering it:
```
 Sep 02 16:12:13 fv-az133-240 systemd[1]: Started Pulp Operator.
  Sep 02 16:12:13 fv-az133-240 pulp[8756]: ts=2022-09-02T16:12:13Z level="\u001b[34mINFO\u001b[0m" logger=controller-runtime.metrics msg="Metrics server is starting to listen" addr=:8080
  Sep 02 16:12:13 fv-az133-240 pulp[8756]: ts=2022-09-02T16:12:13Z level="\u001b[34mINFO\u001b[0m" logger=setup msg="starting manager"
  Sep 02 16:12:13 fv-az133-240 pulp[8756]: ts=2022-09-02T16:12:13Z level="\u001b[34mINFO\u001b[0m" msg="Starting server" path=/metrics kind=metrics addr=[::]:8080
  Sep 02 16:12:13 fv-az133-240 pulp[8756]: ts=2022-09-02T16:12:13Z level="\u001b[34mINFO\u001b[0m" msg="Starting server" kind="health probe" addr=[::]:8081
  ```
  https://github.com/pulp/pulp-operator/runs/8160019725?check_suite_focus=true#step:15:3576